sqlmesh/core/audit/builtin.py

```diff
@@ -76,3 +83,238 @@
 )
     """,
 )
+
+# relationship(to=some.model, source_column=id, target_column=id)
+relationship_audit = Audit(
+    name="relationship",
+    defaults={"from_condition": exp.null(), "to_condition": exp.null()},
+    query="""
+SELECT
+  child.source_column
+FROM (
+  SELECT @source_column as source_column
+  FROM @this_model
+  WHERE NOT @source_column IS NULL
+  AND @IF(@from_condition IS NOT NULL, @from_condition, 1=1)
+) AS child
+LEFT JOIN (
+  SELECT @target_column AS target_column
+  FROM @to
+  @WHERE(@to_condition IS NOT NULL) @to_condition
+) AS parent
+ON child.source_column = parent.target_column
+WHERE parent.target_column IS NULL
+    """,
+)
+
+# accepted_range(column=age, min_v=0, max_v=100)
+# accepted_range(column=age, min_v=10)
+# accepted_range(column=age, max_v=50)
+accepted_range_audit = Audit(
+    name="accepted_range",
+    defaults={"min_v": exp.null(), "max_v": exp.null(), "inclusive": exp.true()},
+    query="""
+SELECT *
+FROM @this_model
+WHERE
+  1=2
+  OR @IF(@min_v IS NOT NULL AND @inclusive, @column <= @min_v, 1=2)
```

Comment:
same type of stuff here

Reply:
we need to support min_v or max_v of 0 which would be falsey in python. So null makes a better sentinel value, no?